src/vs/workbench/contrib/terminal/test/common/terminalEnvironment.test.ts

```diff
@@ -111,8 +111,7 @@ suite('Workbench - TerminalEnvironment', () => {
 			strictEqual(getLangEnvVariable('zh'), 'zh_CN.UTF-8');
 		});
 		test('should set language variant based on full locale', () => {
-			strictEqual(getLangEnvVariable('en-AU'), 'en_AU.UTF-8');
-			strictEqual(getLangEnvVariable('en-au'), 'en_AU.UTF-8');
```

Comment:
Can we keep these tests?

Reply:
Even `en_DE` causes errors, though, so I'm not sure how valid those tests are.
Previously, `app.getLocale()` would return just `en`, so it'd be processed to `en_US` anyway.
Ref https://github.com/microsoft/vscode/issues/168486

Reply:
Edit: I just confirmed that `app.getLocale()` actually returns `en-GB` on macOS when the system language is `English (India)` or `English (Australia)`. However, `app.getPreferredSystemLanguages()[0]` returns `en-IN`, which doesn't correspond to any locale available on the system when running `locale -a`.